doc/GETTING_STARTED.md

```diff
@@ -0,0 +1,428 @@
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [WebSockets],
+    connEncryption: [SECIO]
+  }
+})
+```
+
+<details><summary>Read More</summary>
+If you want to know more about libp2p connection encryption, you should read the following content:
+
+- https://docs.libp2p.io/concepts/secure-comms
+- https://github.com/libp2p/specs/tree/master/connections
+</details>
+
+#### Multiplexing
+
+With libp2p you are able to create your own protocols so that peers can communicate in a meaningful way to your application. Each protocol works as an independent logical stream and you can use a stream multiplexer, in order to share a common underlying transport medium between streams (single connection). As an example, you can create a chat protocol on top of libp2p and use it to exchange chat messages.
+
+Looking at the [available stream multiplexing](./CONFIGURATION.md#stream-multiplexing) protocols, we are only able to use `libp2p-mplex` for the time being. Bear in mind that future libp2p transports might have `multiplexing` capabilities already built-in (such as `QUIC`).
+
+You can install `libp2p-mplex` and add it to your libp2p node as follows in the next example.
+
+```sh
+npm install libp2p-mplex
+```
+
+```js
+const Libp2p = require('libp2p')
+const WEBSOCKETS = require('libp2p-websockets')
+const SECIO = require('libp2p-secio')
+const MPLEX = require('libp2p-mplex')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [WEBSOCKETS],
+    connEncryption: [SECIO],
+    streamMuxer: [MPLEX]
+  }
+})
+```
+
+Protocols design is similar to an http api, where each protocol has an handler, receives a message and returns a response. If you want to create your own protocol, start by having a look at the [Protocol and Stream Muxing Example](../examples/protocol-and-stream-muxing).
+
+<details><summary>Read More</summary>
+If you want to know more about libp2p stream multiplexing, you should read the following content:
+
+- https://docs.libp2p.io/concepts/stream-multiplexing
+- https://github.com/libp2p/specs/tree/master/connections
+- https://github.com/libp2p/specs/tree/master/mplex
+</details>
+
+#### Running Libp2p
+
+Now that you have configured a [`Transport`][transport], [`Crypto`][crypto] and [**Stream Multiplexer**](streamMuxer) module, you can start your libp2p node. We can start and stop libp2p using the [`libp2p.start()`](./API.md#start) and [`libp2p.stop()`](./API.md#stop) methods.
+
+```js
+const Libp2p = require('libp2p')
+const WebSockets = require('libp2p-websockets')
+const SECIO = require('libp2p-secio')
+const MPLEX = require('libp2p-mplex')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [WebSockets],
+    connEncryption: [SECIO],
+    streamMuxer: [MPLEX]
+  }
+})
+
+// start libp2p
+await node.start()
+
+// stop libp2p
+await node.stop()
+```
+
+If you are not familiar with multiaddrs, you should read the [multiformats/js-multiaddr](https://github.com/multiformats/js-multiaddr) README.
```

Comment:
This seems out of place, not sure where this should go.

Reply:
I think we can just remove it